tests/TestModels/Client.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Spatie\Permission\Tests\TestModels;
+
+use Laravel\Passport\Client as BaseClient;
+use Spatie\Permission\Traits\HasRoles;
+
+class Client extends BaseClient
+{
+    use HasRoles;
```

Comment:
maybe
```php
use Illuminate\Contracts\Auth\Access\Authorizable as AuthorizableContract;
use Illuminate\Foundation\Auth\Access\Authorizable;

class Client extends BaseClient implements AuthorizableContract
{
    use Authorizable;
    use HasRoles;
```
For `$client->can($permissions)`

Reply:
Just updated the code with the suggested changes. But now getting either `403` when `200` is expected or `GuardDoesNotMatch` exception

Reply:
Unfortunate the `getDefaultGuardName` method doesn't appear to be working. Adding a guard name variable does appear to be working